Philips.CodeAnalysis.Common/DiagnosticIds.cs

```diff
@@ -120,5 +120,6 @@ public enum DiagnosticIds
 		AlignNumberOfPlusAndEqualOperators = 2125,
 		AvoidUsingParametersAsTempVariables = 2126,
 		AvoidChangingLoopVariables = 2127,
+		AvoidImplementingDestructors = 2130,

```

Comment:
We should use the term 'finalizer' instead. https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/finalizers

Reply:
Renamed